README.md

```diff
@@ -382,6 +386,13 @@ Simple static plots are generated from the outputs of Counter and DESeq2. These
 - **sample_avg_scatter_by_dge**: A scatter plot comparing all sample groups, with significantly different genes highlighted.
 - **sample_avg_scatter_by_dge_class**: A scatter plot comparing all sample groups, with classes and significantly different genes highlighted.
 
+|                                                                                                    |                                                                                       |
+|:--------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------:|
+| <img src="images/plots/scatter_dge_class.jpeg" width="90%" alt="sample_avg_scatter_by_dge_class"/> | <img src="images/plots/scatter_dge.jpeg" width="90%" alt="rule_chart with 10 rules"/> |
+|      <img src="images/plots/class_charts.jpeg" width="90%" alt="sample_avg_scatter_by_dge"/>       |       <img src="images/plots/rule_charts.jpeg" width="90%" alt="rule_charts"/>        |
+|           <img src="images/plots/len_dist_short.jpeg" width="90%" alt="len_dist 20-30"/>           |     <img src="images/plots/len_dist_long.jpeg" width="90%" alt="len_dist 15-60"/>     |
+
+
 DESeq2 can also produce a standard **PCA plot** from variance stabilizing transformed feature counts. This output is controlled by the `dge_pca_plot` key in the Run Config.
```

Comment:
This isn't a current behavior of tiny-deseq.r. PCA plots are only produced if specifically requested via Run Config (and therefore commandline argument `--pca`). This can easily be changed though.